.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
```

Comment:
Thinking updating this to run on `pull_request`

Reply:
yes please